frontend/__tests__/src/pages/Committees.test.tsx

```diff
@@ -40,6 +40,37 @@ describe('Committees Component', () => {
     })
   })
 
+  test('renders SearchBar, data, and pagination component concurrently after data is loaded', async () => {
+    window.scrollTo = jest.fn()
+    ;(loadData as jest.Mock).mockResolvedValue({
+      ...mockCommitteeData,
+      total_pages: 2,
+    })
+
+    render(<CommitteesPage />)
+
+    const loadingSpinner = screen.getAllByAltText('Loading indicator')
+    await waitFor(() => {
+      expect(loadingSpinner.length).toBeGreaterThan(0)
+      expect(
+        screen.queryByPlaceholderText('Search for OWASP committees...')
+      ).not.toBeInTheDocument()
+      expect(screen.queryByText('Next Page')).not.toBeInTheDocument()
+    })
+
+    await waitFor(() => {
+      expect(screen.getByPlaceholderText('Search for OWASP committees...')).toBeInTheDocument()
+
+      expect(screen.getByText('Committee 1')).toBeInTheDocument()
+
+      const nextPageButton = screen.getByText('Next Page')
+      expect(nextPageButton).toBeInTheDocument()
```

Comment:
Could you make this code be consistent for all pages tests (same syntax for `expects()`)

Reply:
sure !